serpentTools/objects/xsdata.py

```diff
@@ -115,8 +111,80 @@ def hasExpectedData(self):
 
         return True
 
+    def tabulate(self, mts='all', colnames=None):
```

Comment:
I really like this! The DataFrames are great for visualization! Plus, those can go straight to LaTeX should the user want too :100: